pkg/asset/templates/templates.go

```diff
@@ -33,20 +33,15 @@ func (m *Templates) Dependencies() []asset.Asset {
 		&bootkube.KubeSystemConfigmapEtcdServingCA{},
 		&bootkube.KubeSystemConfigmapRootCA{},
 		&bootkube.KubeSystemSecretEtcdClient{},
-		&bootkube.TectonicNamespace{},
 		&bootkube.OpenshiftWebConsoleNamespace{},
 		&bootkube.OpenshiftMachineConfigOperator{},
 		&bootkube.OpenshiftClusterAPINamespace{},
 		&bootkube.OpenshiftServiceCertSignerNamespace{},
 		&bootkube.AppVersionKind{},
```

Comment:
AppVersionKind also needs to be dropped.

Reply:
> AppVersionKind also needs to be dropped.

Done with 212450c -> a73e15c.